CHANGELOG.md

```diff
@@ -9,7 +9,7 @@
 * [ENHANCEMENT] Activity tracker logs now have `component=activity-tracker` label. #3556
 * [ENHANCEMENT] Distributor: remove labels with empty values #2439
 * [ENHANCEMENT] Query-frontend: track query HTTP requests in the Activity Tracker. #3561
-* [ENHANCEMENT] Store-gateway: Add experimental alternate implementation of index-header reader that does not use mmap. This reader is expected to improve stability of the store-gateway. This implementation can be enabled with the flag `-blocks-storage.bucket-store.index-header.stream-reader-enabled`. #3639
+* [ENHANCEMENT] Store-gateway: Add experimental alternate implementation of index-header reader that does not use mmap. This reader is expected to improve stability of the store-gateway. This implementation can be enabled with the flag `-blocks-storage.bucket-store.index-header.stream-reader-enabled`. #3639 #3703
```

Comment:
```suggestion
* [ENHANCEMENT] Store-gateway: Add experimental alternate implementation of index-header reader that does not use the mmap Go package. The index-header reader is expected to improve stability of the store-gateway. You can this implementation with the flag `-blocks-storage.bucket-store.index-header.stream-reader-enabled`. #3639 #3703
```
(I assume mmap refers to a Go package, rather than a command or a system call. Please vet this for accuracy.)

As the reader, I all of a sudden thought that "this reader" was somehow referring to me.

Reply:
`mmap` does refer to a system call. I can clarify that in the message in addition to making "this reader" less ambiguous.

Reply:
I think that if we say _does not use the memory mapping technique_ it would be both accurate and meaningful for the readers.